recipes/libsamplerate/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: libsamplerate
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsamplerate.git
+  git_tag: 548c6761a0348a83c5f6ee68aa1581c855092195
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - autoconf # [not win]
+    - automake # [not win]
+    - libtool # [not win]
+    - pkg-config # [not win]
```

Comment:
Could we have two spaces between selectors and packages? Also, could we get the selectors inline with each other?